tests/app/celery/test_tasks.py

```diff
@@ -961,34 +1007,35 @@ def test_save_letter_saves_letter_to_database(sample_letter_job, mocker):
         'postcode': 'Flob',
     }
     notification_json = _notification_json(
-        template=sample_letter_job.template,
+        template=job.template,
         to='Foo',
         personalisation=personalisation,
-        job_id=sample_letter_job.id,
+        job_id=job.id,
         row_number=1
     )
     notification_id = uuid.uuid4()
     created_at = datetime.utcnow()
 
     save_letter(
-        sample_letter_job.service_id,
+        job.service_id,
         notification_id,
         encryption.encrypt(notification_json),
     )
 
     notification_db = Notification.query.one()
     assert notification_db.id == notification_id
     assert notification_db.to == 'Foo'
-    assert notification_db.job_id == sample_letter_job.id
-    assert notification_db.template_id == sample_letter_job.template.id
-    assert notification_db.template_version == sample_letter_job.template.version
+    assert notification_db.job_id == job.id
+    assert notification_db.template_id == job.template.id
+    assert notification_db.template_version == job.template.version
     assert notification_db.status == 'created'
     assert notification_db.created_at >= created_at
     assert notification_db.notification_type == 'letter'
     assert notification_db.sent_at is None
     assert notification_db.sent_by is None
     assert notification_db.personalisation == personalisation
     assert notification_db.reference == "this-is-random-in-real-life"
+    assert notification_db.reply_to_text == "Address contract"
```

Comment:
Should this contain the personalisation address block? 
- A letter contact and been created
- A letter have been created with a personalisation block
Override the contact with the per notification block?